File: platform/sdk/nodejs/examples/graph/graph-019_proj_attr_out.js
const { GraphClient } = require("../../xuanji-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.projAttrOut("node-person-1");
if (!result.exported || !result.attributes.name) process.exit(1);
console.log("XJ-OK: graph-019_proj_attr_out");
process.exit(0);

File: platform/sdk/python/examples/graph/graph-019_proj_attr_out.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.projAttrOut("node-person-1")
if not result["exported"] or not result["attributes"]["name"]:
    sys.exit(1)
print("XJ-OK: graph-019_proj_attr_out")
sys.exit(0)
